samples/emoji-search/android-views/src/main/kotlin/com/example/redwood/emojisearch/android/views/EmojiSearchApplication.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2024 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.redwood.emojisearch.android.views
+
+import android.app.Application
+import android.content.Context
+import coil3.ImageLoader
+import coil3.SingletonImageLoader
+import coil3.annotation.ExperimentalCoilApi
+import coil3.fetch.NetworkFetcher
+
+class EmojiSearchApplication : Application(), SingletonImageLoader.Factory {
+  @OptIn(ExperimentalCoilApi::class)
+  override fun newImageLoader(context: Context): ImageLoader {
+    return ImageLoader.Builder(context)
+      .components {
+        add(NetworkFetcher.Factory())
```

Comment:
Network URL support currently has to be added manually as it was pulled out of `coil-core`. Ktor finds imported engines at runtime, which could also be done by Coil, but I'm not sure it's a good idea.